strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/proxied/ProxyRepositoryArtifactResolver.java

```diff
@@ -77,7 +77,17 @@ public InputStream getInputStream(final String storageId,
                                                                                          remoteRepository.getPassword()))
         {
             final LayoutProvider layoutProvider = layoutProviderRegistry.getProvider(repository.getLayout());
-            final URI resource = layoutProvider.resolveResource(repository, path);
+
+            URI resource;
+            try
+            {
+                resource = layoutProvider.resolveResource(repository, path);
+            }
+            catch(IllegalArgumentException e)
```

Comment:
`catch(IllegalArgumentException e)`

Could you please always format the code fragments you add by the code formatter https://github.com/strongbox/strongbox/wiki/Coding-Convention#code-styles ?